ts/store/actions/authentication.ts

```diff
@@ -65,6 +69,18 @@ export const resetAuthenticationState = createStandardAction(
   "RESET_AUTHENTICATION_STATE"
 )();
 
+export const checkCurrentSession = createStandardAction(
```

Comment:
An improvement you could do on creating action is about using **createAsyncAction**
createAsyncAction does what you did but with a factory function
Then you can use **_checkCurrentSession_** for each case

- checkCurrentSession.request()
- checkCurrentSession.success({isSessionValid: true/false})
- checkCurrentSession.failure(new Error("ops"))

```typescript
export const checkCurrentSession = createAsyncAction(
  "CHECK_CURRENT_SESSION_REQUEST",
  "CHECK_CURRENT_SESSION_SUCCESS",
  "CHECK_CURRENT_SESSION_FAILURE"
)<void, CheckSessionResult, Error>();
```

Reply:
Added Tests and asynchronous actions on 09fddb0